ui/pages/confirmations/components/alerts/alert-modal/alert-modal.tsx

```diff
@@ -49,6 +49,11 @@ export type AlertModalProps = {
     /** The variant of the custom button. */
     variant?: string;
   };
+  /**
+   * The navigation component passed when more exists more than one alert.
+   * It override `startAccessory` of ModalHeaderDefault and by default no navigation button is present.
+   */
+  multipleAlerts?: React.ReactNode;
```

Comment:
Rather than coupling this to the concept of multiple alerts, should we just call this `headerStartAccessory` and update the comment also?